scripts/chime7/pipeline/chime_config.yaml

```diff
@@ -16,7 +16,7 @@ subsets:
 
 num_workers: 16
 sample_rate: 16000
-batch_size: 8  # batch size for diarization and ASR
+batch_size: 1  # batch size for diarization and ASR
```

Comment:
@tango4j we need separate batch size for major steps in the pipeline. 
ASR needs batch size 1 or I get OOM on dipco.

Reply:
Added